### PR TITLE
Kavs drinks

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
Makes  many new department and race drinks sprited and designed by Kavrick

### CODE REVIEW

File: code/modules/reagents/reagents/food-Drinks.dm [hunk 1]
```diff
@@ -2937,3 +2956,201 @@
...
+
+	glass_unique_appearance = TRUE
+	glass_icon_state = "arcscrewdriver"
+	glass_name = "Arc screwdriver"
+	glass_desc = "For one a regular one doesn't cut it."
+
+/datum/reagent/ethanol/whiskey_rocks
+	name = "Whiskey Rocks"
+	id = "whiskey_rocks"
+	description = "Whiske but for people that like things cold and waterd down."
```
Comment (cdb-is-not-good) [inline code/modules/reagents/reagents/food-Drinks.dm:3044]:
```suggestion
	description = "Whiskey but for people that like things cold and watered down."
```

File: code/modules/reagents/reagents/food-Drinks.dm [hunk 2]
```diff
@@ -2710,16 +2710,33 @@
...
 	strength = 25
 
+	glass_unique_appearance = TRUE
 	glass_name = "Daiquiri"
 	glass_desc = "Refreshing rum and citrus. Time for a tropical get away."
 
+/datum/reagent/ethanol/daiquiri_strawberry
+	name = "Daiquiri Pink"
+	id = "daiquiri_strawberry"
+	description = "Refreshing rum and strawberry juice. Time for a tropical get away, even if its to fruity."
```
Comment (cdb-is-not-good) [inline code/modules/reagents/reagents/food-Drinks.dm:2723]:
```suggestion
	description = "Refreshing rum and strawberry juice. Time for a tropical get away, even if its too fruity."
```

File: code/modules/reagents/reagents/food-Drinks.dm [hunk 3]
```diff
@@ -2710,16 +2710,33 @@
...
+	taste_description = "refreshing strawberry and rum"
+	color = "#d1ff49d0" // rgb(209, 255, 73
+	glass_unique_appearance = TRUE
+	glass_icon_state = "daiquiristrawberry"
+	strength = 23 //Washed down with FLAVOUR
+
+	glass_name = "Daiquiri Pink"
+	glass_desc = "Refreshing rum and strawberry juice. Time for a tropical get away, even if its to fruity."
```
Comment (cdb-is-not-good) [inline code/modules/reagents/reagents/food-Drinks.dm:2731]:
```suggestion
	glass_desc = "Refreshing rum and strawberry juice. Time for a tropical get away, even if its too fruity."
```

File: code/modules/reagents/reagents/food-Drinks.dm (hunk 1 above)
Comment (ThePainkiller) [reply code/modules/reagents/reagents/food-Drinks.dm:3044]: Should...should also be called "Whiskey IN the rocks"....unless these are literally solid rocks made out of whiskey...? 😂

File: code/modules/reagents/reagents/food-Drinks.dm [hunk 4]
```diff
@@ -2937,3 +2956,201 @@
...
+	glass_name = "Southern Sake"
+	glass_desc = "Best served warm and in a shallow bowl for lapping."
+
+/datum/reagent/ethanol/nothern_mead
+	name = "Nothern Mead"
+	id = "nothern_mead"
+	description = "Said to make even the strongest of accents all the more unintelligible."
+	taste_description = "robust hops and carbon"
+	color = "#E0CE8A" // rgb(88, 81, 54)
+	strength = 140 //stronk
```
Comment (ThePainkiller) [inline code/modules/reagents/reagents/food-Drinks.dm:2982]: _Ackshually_, `strength` works inversely as tg's `boozepwr` code wise: [The lower the value, the stronger the drink](https://github.com/sojourn-13/sojourn-station/blob/716e363a731bef79140d21c84a4447f1fea36800/code/modules/reagents/reagents/dispenser.dm#L111), hence why Absinthe and Moonshine will get you drunker easier. If you want it to be strong, make it between 10 (baseline ethanol) and 30.

File: code/modules/reagents/reagents/food-Drinks.dm [hunk 5]
```diff
@@ -2937,3 +2956,201 @@
...
+	color = "#E0CE8A" // rgb(88, 81, 54)
+	strength = 60
+
+	glass_unique_appearance = TRUE
+	glass_icon_state = "posicell"
+	glass_name = "Posicell"
+	glass_desc = "The only drink that is shockingly annonying to make in and out of house."
+
+/datum/reagent/ethanol/nickle_n_dime
+	name = "Nickle And Dime"
```
Comment (ThePainkiller) [inline code/modules/reagents/reagents/food-Drinks.dm:3016]:
```suggestion
	name = "Nickel And Dime"
```

File: code/modules/reagents/reagents/food-Drinks.dm [hunk 6]
```diff
@@ -2937,3 +2956,201 @@
...
+	id = "posicell"
+	description = "The only drink that is shockingly annonying to make in and out of house."
+	taste_description = "rouge hardware and annoyed labcoats"
+	color = "#E0CE8A" // rgb(88, 81, 54)
+	strength = 60
+
+	glass_unique_appearance = TRUE
+	glass_icon_state = "posicell"
+	glass_name = "Posicell"
+	glass_desc = "The only drink that is shockingly annonying to make in and out of house."
```
Comment (ThePainkiller) [inline code/modules/reagents/reagents/food-Drinks.dm:3013]:
```suggestion
	glass_desc = "The only drink that is shockingly annoying to make in and out of the house."
```

File: code/modules/reagents/reagents/food-Drinks.dm [hunk 7]
```diff
@@ -2937,3 +2956,201 @@
...
+
+	glass_unique_appearance = TRUE
+	glass_icon_state = "lifeline"
+	glass_name = "Life Line"
+	glass_desc = "A perfect drink for mass casualties."
+
+/datum/reagent/ethanol/posicell
+	name = "Posicell"
+	id = "posicell"
+	description = "The only drink that is shockingly annonying to make in and out of house."
```
Comment (ThePainkiller) [inline code/modules/reagents/reagents/food-Drinks.dm:3005]:
```suggestion
	description = "The only drink that is shockingly annoying to make in and out of the house."
```

File: code/modules/reagents/reagents/food-Drinks.dm (hunk 2 above)
Comment (ThePainkiller) [reply code/modules/reagents/reagents/food-Drinks.dm:2723]:
```suggestion
	description = "Refreshing rum and strawberry juice. Time for a tropical getaway, even if it's too fruity."
```

File: code/modules/reagents/reagents/food-Drinks.dm [hunk 8]
```diff
@@ -2710,16 +2710,33 @@
 	description = "Refreshing rum and citrus. Time for a tropical get away."
 	taste_description = "refreshing citrus and rum"
 	color = "#d1ff49d0" // rgb(209, 255, 73
+	glass_icon_state = "daiquiri"
 	strength = 25
 
+	glass_unique_appearance = TRUE
 	glass_name = "Daiquiri"
 	glass_desc = "Refreshing rum and citrus. Time for a tropical get away."
```
Comment (ThePainkiller) [inline code/modules/reagents/reagents/food-Drinks.dm:2718]:
```suggestion
	glass_desc = "Refreshing rum and citrus. Time for a tropical getaway."
```

File: code/modules/reagents/reagents/food-Drinks.dm [hunk 9]
```diff
@@ -2937,3 +2956,201 @@
...
+
+	glass_unique_appearance = TRUE
+	glass_icon_state = "nickleanddime"
+	glass_name = "Nickle And Dime"
+	glass_desc = "For those with a rich taste."
+
+/datum/reagent/ethanol/arcscrewdriver
+	name = "Arc screwdriver"
+	id = "arcscrewdriver"
+	description = "For one a regular one doesn't cut it."
```
Comment (ThePainkiller) [inline code/modules/reagents/reagents/food-Drinks.dm:3031]:
```suggestion
	description = "For when a regular one doesn't cut it."
```

File: code/modules/reagents/reagents/food-Drinks.dm [hunk 10]
```diff
@@ -2937,3 +2956,201 @@
...
+	taste_description = "electrooptic discussion"
+	color = "#E0CE8A" // rgb(88, 81, 54)
+	strength = 75
+
+	glass_unique_appearance = TRUE
+	glass_icon_state = "arcscrewdriver"
+	glass_name = "Arc screwdriver"
+	glass_desc = "For one a regular one doesn't cut it."
```
Comment (ThePainkiller) [inline code/modules/reagents/reagents/food-Drinks.dm:3039]:
```suggestion
	glass_desc = "For when a regular one doesn't cut it."
```

File: code/modules/reagents/reagents/food-Drinks.dm [hunk 11]
```diff
@@ -2937,3 +2956,201 @@
...
+
+/datum/reagent/ethanol/whiskey_rocks
+	name = "Whiskey Rocks"
```
Comment (ThePainkiller) [inline code/modules/reagents/reagents/food-Drinks.dm:3042]:
```suggestion
	name = "Whiskey on the rocks"
```

File: code/modules/reagents/recipes/recipes_food_drinks.dm [hunk 12]
```diff
@@ -785,3 +790,83 @@
...
+	required_reagents = list("vodka" = 3, "lemon_lime" = 3, MATERIAL_GOLD = 1)
+	result_amount = 3
+
+/datum/chemical_reaction/drinks/arcscrewdriver
+	result = "arcscrewdriver"
+	required_reagents = list("screwdrivercocktail" = 4, "carbon" = 1, "iron" = 1, MATERIAL_PLASMA = 1)
+	result_amount = 5 //carbon iron and plasma makes plasteel materal wise
+
+/datum/chemical_reaction/drinks/whiske_rocks
+	result = "whiske_rocks"
```
Comment (ThePainkiller) [inline code/modules/reagents/recipes/recipes_food_drinks.dm:830]:
```suggestion
	result = "whiskey_rocks"
```

File: code/modules/reagents/recipes/recipes_food_drinks.dm [hunk 13]
```diff
@@ -785,3 +790,83 @@
...
+	required_reagents = list("whiskey" = 2, "ice" = 1)
+	result_amount = 4 //Is is dence
+
+/datum/chemical_reaction/drinks/nior
+	result = "nior"
+	required_reagents = list("whiske_rocks" = 2, "rum" = 1)
```
Comment (ThePainkiller) [inline code/modules/reagents/recipes/recipes_food_drinks.dm:836]:
```suggestion
	required_reagents = list("whiskey_rocks" = 2, "rum" = 1)
```

File: code/modules/reagents/recipes/recipes_food_drinks.dm [hunk 14]
```diff
@@ -785,3 +790,83 @@
...
+
+/datum/chemical_reaction/drinks/arcscrewdriver
+	result = "arcscrewdriver"
+	required_reagents = list("screwdrivercocktail" = 4, "carbon" = 1, "iron" = 1, MATERIAL_PLASMA = 1)
+	result_amount = 5 //carbon iron and plasma makes plasteel materal wise
+
+/datum/chemical_reaction/drinks/whiske_rocks
+	result = "whiske_rocks"
+	required_reagents = list("whiskey" = 2, "ice" = 1)
+	result_amount = 4 //Is is dence
```
Comment (ThePainkiller) [inline code/modules/reagents/recipes/recipes_food_drinks.dm:832]:
```suggestion
	result_amount = 3 //Is is dence
```

File: code/modules/reagents/reagents/food-Drinks.dm [hunk 15]
```diff
@@ -2937,3 +2956,201 @@
...
+	glass_unique_appearance = TRUE
+	glass_icon_state = "buckshot"
+	glass_name = "Rusty Special"
+	glass_desc = "Both Barrels."
+
+/datum/reagent/ethanol/western_escargot
+	name = "Western Escargot"
+	id = "western_escargot"
+	description = "The kinda slugs a free man can appreciate."
+	taste_description = "an explosion of flavours and a nice kick after taste"
```
Comment (ThePainkiller) [inline code/modules/reagents/reagents/food-Drinks.dm:3123]:
```suggestion
	taste_description = "an explosion of flavors and a nice kick on its aftertaste"
```

File: code/modules/reagents/reagents/food-Drinks.dm [hunk 16]
```diff
@@ -2937,3 +2956,201 @@
...
+	glass_unique_appearance = TRUE
+	glass_icon_state = "marquadrink"
+	glass_name = "Ocular"
+	glass_desc = "Don't make eye contact, a favorite of the Mar qua."
+
+/datum/reagent/ethanol/saint_pio
+	name = "Saint Pio"
+	id = "saint_pio"
+	description = "An absolutely perfect choice."
+	taste_description = "crusade of flavours"
```
Comment (ThePainkiller) [inline code/modules/reagents/reagents/food-Drinks.dm:3097]:
```suggestion
	taste_description = "a crusade of flavors"
```

File: code/modules/reagents/reagents/food-Drinks.dm [hunk 17]
```diff
@@ -2937,3 +2956,201 @@
...
+	glass_unique_appearance = TRUE
+	glass_icon_state = "nior"
+	glass_name = "Nior"
+	glass_desc = "For when things aren't always black and white."
+
+/datum/reagent/ethanol/duress
+	name = "Duress"
+	id = "duress"
+	description = "A good way to get people talking."
+	taste_description = "stunglove fiber"
```
Comment (ThePainkiller) [inline code/modules/reagents/reagents/food-Drinks.dm:3071]:
```suggestion
	taste_description = "stunglove fibers"
```

File: code/modules/reagents/reagents/food-Drinks.dm [hunk 18]
```diff
@@ -2937,3 +2956,201 @@
...
+	color = "#E0CE8A" // rgb(88, 81, 54)
+	strength = 23
+
+	glass_unique_appearance = TRUE
+	glass_icon_state = "noirbase"
+	glass_name = "Whiskey Rocks"
+	glass_desc = "Whiskey but for people that like things cold and waterd down."
+
+/datum/reagent/ethanol/nior
+	name = "Nior"
```
Comment (ThePainkiller) [inline code/modules/reagents/reagents/food-Drinks.dm:3055]:
```suggestion
	name = "Noir"
```

File: code/modules/reagents/reagents/food-Drinks.dm [hunk 19]
```diff
@@ -2937,3 +2956,201 @@
...
+	id = "nior"
+	description = "For when things aren't always black and white."
+	taste_description = "a cold case"
+	color = "#E0CE8A" // rgb(88, 81, 54)
+	strength = 38 //.38 subnose if only!
+
+	glass_unique_appearance = TRUE
+	glass_icon_state = "nior"
+	glass_name = "Nior"
```
Comment (ThePainkiller) [inline code/modules/reagents/reagents/food-Drinks.dm:3064]:
```suggestion
	glass_name = "Noir"
```

**Review by ThePainkiller — CHANGES_REQUESTED** [review]
Minor typo fixes + actually making Whiskey on the rocks and Noir mixable

**Pull request merged**